test/Datadog.Trace.ClrProfiler.Native.Tests/metadata_builder_tests.cpp

```diff
@@ -0,0 +1,90 @@
+#include "pch.h"
+
+#include "../../src/Datadog.Trace.ClrProfiler.Native/iterators.h"
+#include "../../src/Datadog.Trace.ClrProfiler.Native/metadata_builder.h"
+
+class MetadataBuilderTest : public ::testing::Test {
+ protected:
+  ModuleMetadata* moduleMetadata_;
+  MetadataBuilder* metadataBuilder_;
+
+  void SetUp() override {
+    ICLRMetaHost* metahost = NULL;
```

Comment:
Use the standard `nullptr` instead of `NULL`? (I know the current C++ code sets a bad example, and that's my fault, but we should try to stick to modern C++ going forwards.)